### ISSUE TITLE
Heroku feedback gives a 404 Error for Github

### ISSUE THREAD

**Issue by harraton** [mozilla/firefox-voice#875]
"Also, I just submitted a bug via the Heroku feedback app, and it had a link to click to see my issue on GitHub. That link went to a 404 error."

https://github.com/mozilla/firefox_voice_community/issues/231

**Comment by awallin** [comment]
@harraton Yes, this is an issue @couci and @MichaelKohler are aware of and looking to fix. I've stressed the urgency as we can no longer collect feedback from people using Firefox Voice.

I believe the issues involves the bot creation of anonymous issues in Github?
